### DISCUSSION TITLE
Suggest GitHub to provide mobile app

### DISCUSSION THREAD

**Discussion by longdafeng** [community/community#52243] [Mobile] [answered]
### Select Topic Area

Question

### Body

I wish there is one GitHub mobile app, which I can check my repo's issue or pull request.

Right now, logining github on mobile phone's web browser is pretty complicated.

At the same time, I often recommend my project https://github.com/oceanbase/oceanbase to my friend, if there is one mobile phone app, my friends is easy to add my project to favoriate.

**Comment by jricciardi** [comment, marked as answer]
Hi there @longdafeng! Good news - there actually is a GitHub mobile app. You can find more info and a download link here: https://github.com/mobile

I'm also going to move this post to the `Mobile` Category of this Discussions forum. If you have any feedback about the mobile app, that would be the place to add your voice and upvote any user-requested features 😄
